packages/lib/models/Tag.ts

```diff
@@ -216,4 +216,20 @@ export default class Tag extends BaseItem {
 			return tag;
 		});
 	}
+
+	static sortTags(tags: TagEntity[]) {
+		if (tags[0].title) {
```

Comment:
I don't think you need the condition anymore

Reply:
oops!! will remove it asap